opendbc/car/tests/routes.py

```diff
@@ -0,0 +1,295 @@
+from typing import NamedTuple
+
+from opendbc.car.chrysler.values import CAR as CHRYSLER
+from opendbc.car.gm.values import CAR as GM
+from opendbc.car.ford.values import CAR as FORD
+from opendbc.car.honda.values import CAR as HONDA
+from opendbc.car.hyundai.values import CAR as HYUNDAI
+from opendbc.car.nissan.values import CAR as NISSAN
+from opendbc.car.mazda.values import CAR as MAZDA
+from opendbc.car.mock.values import CAR as MOCK
+from opendbc.car.subaru.values import CAR as SUBARU
+from opendbc.car.toyota.values import CAR as TOYOTA
+from opendbc.car.values import Platform
+from opendbc.car.volkswagen.values import CAR as VOLKSWAGEN
+from opendbc.car.body.values import CAR as COMMA
```

Comment:
@sshane what do you think of collapsing this into just `CAR`? one small step towards support groups...